keyboards/freyr/readme.md

```diff
@@ -0,0 +1,14 @@
+# freyr
+
+![Freyr](https://i.imgur.com/MeOrzoI.jpg)
+
+Freyr is a tenkeyless keyboard,
```

Comment:
If this matches the community TKL layout, you may want to consider adding community layout support for the board.

https://github.com/qmk/qmk_firmware/tree/master/layouts/default/tkl_iso
https://github.com/qmk/qmk_firmware/tree/master/layouts/default/tkl_ansi


https://docs.qmk.fm/#/feature_layouts

Reply:
okie, i added 2 layouts